### PR TITLE
nvdimm_readonly: Add new case to test readonly option

### PR DESCRIPTION
**Pull request by huangyum**
Signed-off-by: Yumei Huang <yuhuang@redhat.com>
ID: 1972595
Depends on https://github.com/avocado-framework/avocado-vt/pull/3103.

### CODE REVIEW

**Comment by huangyum** [comment]
Test would fail due to qemu bug1963845, but the qemu cmdline generated is expected. 
@yanan-fu Would you pls help review, thanks.

File: qemu/tests/cfg/nvdimm.cfg [hunk 1]
```diff
@@ -111,6 +111,17 @@
                 - align_illegal:
                     align_mem = 1.5G
                     error_msg = "alignment 0x60000000 must be a power of two"
+        - nvdimm_readonly:
+            readonly_mem_mem1 = on
+            variants:
+                - unarmed_on:
+                    type = nvdimm_negative
+                    unarmed_dimm = on
```
Comment (yanan-fu) [inline qemu/tests/cfg/nvdimm.cfg:119]: `unarmed_dimm_mem1` is more precise.

File: qemu/tests/cfg/nvdimm.cfg [hunk 2]
```diff
@@ -111,6 +111,17 @@
...
+                    start_vm = no
+                    error_msg = "'unarmed' property must be off since memdev mem-${mem_devs} is read-only"
+                - unarmed_off:
```
Comment (yanan-fu) [inline qemu/tests/cfg/nvdimm.cfg:122]: trailing whitespace at L122

File: qemu/tests/cfg/nvdimm.cfg (hunk 1 above)
Comment (huangyum) [reply qemu/tests/cfg/nvdimm.cfg]: Thanks, updated.

**Comment by yanan-fu** [comment]
Ack

**Review by yanan-fu — APPROVED** [review]

**Pull request merged**